spec/lib/bamboozled/employee_spec.rb

```diff
@@ -112,5 +112,25 @@
       expect(location).to eq "https://api.bamboohr.com/api/gateway.php/alphasights/v1/employees/44259"
     end
   end
+
+  describe "#update" do
+    it 'updates an employee in BambooHR' do
+      xml = YAML.load_file('spec/fixtures/update_employee_xml.yml')
+      response = File.new('spec/fixtures/update_employee_response.json')
+      details = JSON.parse(File.read('spec/fixtures/update_employee_details.json'))
+
+      stub_request(:post, "https://x:x@api.bamboohr.com/api/gateway.php/x/v1/employees/1234").
+        with(xml).to_return(response)
+
+      employee = @client.employee.update(bamboo_id: '1234', employee_details: details)
```

Comment:
Line is too long. [86/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.